app/views/movies/_movie.html.erb

```diff
@@ -1,22 +1,16 @@
 <div id="<%= dom_id movie %>">
-  <p>
-    <strong>Title:</strong>
-    <%= movie.title %>
-  </p>
-  <p>
-    <strong>Description:</strong>
-    <%= movie.description %>
-  </p>
-
-  <p>
-    <strong>Time:</strong>
-    <%= movie.duration %>
-  </p>
-
-  <% @screenings.each do |screening| %>
-    <% if screening.movie_id == movie.id %>
-      <p> <%= link_to "#{screening.start_time.strftime("%d/%m/%Y %H:%M")}", new_screening_reservation_url(screening.id) %>
-    <%end%>
-  <%end%>
-  
-</div>
+  <div class="card" style="width: 30rem;">
+    <div class="card-body">
+      <h5 class="card-title"><%= movie.title %></h5>
+      <h6 class="card-subtitle mb-2 text-muted"><%= movie.duration %> min</h6>
+      <p class="card-text"><%= movie.description %></p>
+      <div class="d-flex flex-row">
+      <% @screenings.each do |screening| %>
+        <% if screening.movie_id == movie.id && which_day?(screening) %>
```

Comment:
Instead of `@screening = Screening.all` which is a bad query looking at the performance, you can go with `movie.screenings` :). Both queries `Screening.all` should be removed from `index` and `show` actions